### PR TITLE
return: add page

### PR DESCRIPTION
**Pull request by Managor**
#10584

- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/return.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# return
+
+> Exit out of a function or a script if run with `source`.
+> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-return>.
+
+- Exit out of a function prematurely:
```
Comment (gutjuri) [inline pages/common/return.md:6]:
```suggestion
> Exit a function or a script if run with `source`.
> More information: <https://www.gnu.org/software/bash/manual/bash.html#index-return>.

- Exit a function prematurely:
```

File: pages/common/return.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`{{func_name}}() { {{echo "This is reached"}}; return; {{echo "This is not"}}; }`
+
+- Specify function return value:
```
Comment (gutjuri) [inline pages/common/return.md:10]:
```suggestion
- Specify the function's return value:
```

**Review by gutjuri — APPROVED** [review]

File: pages/common/return.md (hunk 2 above)
Comment (Managor) [reply pages/common/return.md:10]: I feel my wording is not wrong and it feels more punctual. I'll leave it as is unless you have convincing arguments.

Comment (gutjuri) [reply pages/common/return.md:10]: At least the `the` is needed in order to have a correct English sentence. 

The `'s` is personal preference. Writing `Specify the function return value` implies that `function return value` is a commonly used name. I personally think that `Specify the function's return value` reads more smoothly, but I'll approve both if you like the version without the `'s` better.

Comment (Managor) [reply pages/common/return.md:10]: The pains of being an ESL.

**Pull request merged**